exonum/src/helpers/fabric/details.rs

```diff
@@ -32,7 +33,6 @@ use super::{
 };
 use api::backends::actix::AllowOrigin;
 use blockchain::{config::ValidatorKeys, GenesisConfig};
-use crypto;
```

Comment:
Why move it? `crypto` is a module in `exonum` crate. Ditto for other places.

Reply:
Seems, fmt moved it there